python/lsst/ts/ofc/controllers/base_controller.py

```diff
@@ -173,44 +199,64 @@ def aggregated_state(self):
         """
         return self.dof_state[self.ofc_data.dof_idx]
 
+    def reset_history(self) -> None:
+        """Reset the history of the controller."""
+        self.dof_state0 = self.dof_state
```

Comment:
You might need to do the following instead as the `self.dof_state` is an object:

```
self.dof_state0 = self.dof_state.copy()
```

Please make sure you have a unit test to make sure the `id(dof_state0)` and `id(dof_state)` are different.